### ISSUE TITLE
Deal making with Cron schedule

### ISSUE THREAD

**Issue by xinaxu** [data-preservation-programs/singularity#108]
## Prerequisite
- [ ] Have some dataset prepared
- [ ] Have the dataset associated with a wallet #106 
- [ ] #110 

# Test List
- [ ] Run deal making service `singularity run dealmaker`
- [ ] Run deal tracking service `singularity run deal-tracker`
- [ ] Use `singularity deal schedule create` to send deals from a dataset to a single storage provider according to some schedule 
  - [ ] Do not set any options in "Scheduling". It will be covered in #108
  - [ ] No need to test different combination of options in "Deal Proposal" as it is already covered in #107
  - [ ] Test all options under `Scheduling`, i.e. 15 deals per hour is `--schedule-cron "0 * * * *" --schedule-deal-number 15`
- [ ] `singularity deal schedule list` to list all current schedules
  - [ ] Will never be marked complete as the schedule will monitor slashed deals or those that needs renewal
- [ ] `singularity deal pause` to pause the current schedule - takes a few seconds to take effect
- [ ] `singularity deal resume` to resume the current schedule - takes a few seconds to take effect
- [ ] After the storage provider have published / sealed some deals, check the deal status using `singularity deal list`

**Comment by xinaxu** [comment]
We should also test schedule change after #114 is done

**Comment by xinaxu** [comment]
Covered in unit test
